srcpkgs/greetd/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'greetd'
+pkgname=greetd
+version=0.7.0
+revision=1
+build_style=cargo
+conf_files="/etc/greetd/greetd.conf"
+hostmakedepends="scdoc"
+makedepends="pam-devel"
+short_desc="Minimal and flexible login manager daemon"
+maintainer="travankor <travankor@tuta.io>"
+license="GPL-3.0-or-later"
+homepage="https://git.sr.ht/~kennylevinsen/greetd"
+distfiles="https://git.sr.ht/~kennylevinsen/greetd/archive/${version}.tar.gz"
+checksum=c84214490479f291ed3f27424e6c020a9f3115f5745c90a05f7508999b1b69a3
+
+system_accounts="_greeter"
+_greeter_groups="video"
+
+pre_build() {
+	# Void accounts are prefixed with underscore
+	vsed -i "16s/greeter/_greeter/" config.toml
+
+	for i in man/*.scd; do
+		scdoc < "$i" > "${i:: -4}"
+	done
+
+	for f in man/*-[0-9]; do
+		num="${f: -1}"
+		mv "$f" "${f/-$num/.$num}"
+	done
+}
+
+do_install() {
+	vbin "target/${RUST_TARGET}/release/agreety"
+	vbin "target/${RUST_TARGET}/release/fakegreet"
+	vbin "target/${RUST_TARGET}/release/greetd"
+	vinstall ${FILESDIR}/greetd.pam 644 etc/pam.d/
+	vinstall config.toml 644 etc/greetd/ greetd.conf
+	vsv greetd
+
+	for i in man/*.[0-9]; do
+		vman "$i"
+	done
+}
+
+post_install() {
+	rm -f "${DESTDIR}"/usr/.crates.toml
+	rm -f "${DESTDIR}"/usr/.crates2.json
```

Comment:
How is this ever copied there??

Reply:
This is normally part of `do_install()` for the cargo build style, but I moved it to `post_install()` since there is a custom `do_install()`.